pylint/checkers/variables.py

```diff
@@ -520,6 +527,13 @@ def _has_locals_call_after_node(stmt: nodes.NodeNG, scope: nodes.FunctionDef) ->
         "Emitted when an index used on an iterable goes beyond the length of that "
         "iterable.",
     ),
+    "W0644": (
+        "Possible unbalanced dict unpacking with "
+        "sequence%s: "
```

Comment:
```suggestion
        "sequence %s: "
```

Reply:
I assume this goes along with your comment

> ... *Possible unbalanced tuple unpacking with sequence%s: looks very weird.

1. this is the same as above unbalanced-tuple... does
2. current output is

`Possible unbalanced dict unpacking with sequence: left side`
while if I make the suggested change it would be
`Possible unbalanced dict unpacking with sequence : left side`

is that your intention? Idk why we would add a space

Reply:
I just noticed that the help message looks off. Imo, the help message should read:
`Possible unbalanced dict unpacking with %s: ...`